ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -54,7 +54,7 @@ export default Route.extend({
   beforeModel() {
     let secret = this.secretParam();
     let backend = this.enginePathParam();
-    let { tab } = this.paramsFor('vault.cluster.secrets.backend');
+    let { tab } = this.paramsFor('vault.cluster.secrets.backend.list-root');
```

Comment:
We should assign this to a separate variable since it was previously used in [PR 6294](https://github.com/hashicorp/vault/pull/6294/files)

Reply:
Checked this, and getting `tab` from `list-root` still works for the other scenario it's used in, which is PKI